src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Alerting concepts and terms
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: 'Some basic terms and concepts for New Relic alerting and applied intelligence.'
+redirects:
+  - /docs/enable-new-relic-ai-0
+  - /docs/enable-new-relic-ai
+  - /docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence
+  - /docs/alerts-applied-intelligence/incident-intelligence/get-started-incident-intelligence
+  - /docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/eu-us-datacenters-incident-intelligence
+---
+
+import alertConceptsDiagram from 'images/alerting-concepts-diagram.png'
+
+To make the best use of New Relic alerting and applied intelligence features, it will help you to understand some basic terms and concepts. In the following explanation, the bolded words are important New Relic terms.
+
+<img
+  title="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  alt="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  src={nraiDecisionsIndex}
+/>
+
+You can set up alerts for your telemetry data by configuring alert **conditions**, which define what will constitute a violation and trigger an **incident**. For example, you might want to create an alert condition that opens an incident if a host's CPU goes above 80%. Multiple incidents can be aggregated into an **issue**. 
```

Comment:
> Multiple incidents can be aggregated into an **issue**. 

One or multiple incidents will be aggregated into an issue -- no incident is allowed to roam free, they are always rolled up into issues, even if it's only one incident in there.